dev/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.0
+ENV PYTHONUNBUFFERED 1
+
+# Update
+RUN apt update -y && apt upgrade -y && apt install gcc libffi-dev wget git -y
+
+EXPOSE 11500
+
+RUN mkdir /code
+WORKDIR /code
+ADD . /code
+
+# install reqs and solc
+RUN pip install --upgrade pip \
+    && pip install pipenv \
+    && pip3 install -r dev-requirements.txt --src /usr/local/src \
```

Comment:
Are development requirements required to run this Dockerfile?  Which dev dependencies are being used?

Reply:
Yes because the intention is to be able to _do_ development with this setup.  Running the test suite etc.  

I am not sure which are being used.  You would know more!

Reply:
It depends how you envision this fleet being employed.  Most of the `dev-requirements` are aimed at running tests.

Reply:
Point being, unless you aim to run tests *inside* the containers, then they dont necessarily need to have dev-dependencies installed; and you can still run tests that *connect* to this docker fleet fleet (anyone correct me if I'm wrong - I'm still trying to think of a solid advantage I'm missing out on).

I also dont know that there is a big *disadvantage* to using dev-dependencies here - However if we can share some common dev and deploy docker works, that seems efficient. :-)

Reply:
Also - these containers use a local volume binding - so I can update the application code running inside the containers?

Reply:
yes exactly.  when you change the local code, the containers will be running that code.  Figuring that out was some of the hard part of this!